.npmignore

```diff
@@ -1,53 +1,2 @@
-# Logs
-logs
-*.log
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
-
-# Coverage directory used by tools like istanbul
-coverage
-
-# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-# Compiled binary addons (http://nodejs.org/api/addons.html)
-build/Release
-
-# Dependency directory
-# Commenting this out is preferred by some people, see
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
-node_modules
-
-# Users Environment Variables
-.lock-wscript
-.tsdrc
-
-#IntelliJ configuration files
-.idea
-
-/demo
-/test
-Thumbs.db
-.DS_Store
-*.yml
-!*.d.ts
-/src
-*.spec.*
-*.e2e.*
-CONTRIBUTING.md
-karma-shim.js
-karma.conf.js
-protractor.conf.js
-tsconfig.json
-tslint.json
-typedoc.json
-webpack.config.js
-.travis.yml
-.jshintrc
-.editorconfig
+/*
+!/dist
```

Comment:
To clarify, npm will ALWAYS include `readme`(and variations) `license` and variations `package.json`